docs/source/using/profiles.adoc

```diff
@@ -0,0 +1,35 @@
+= Minishift Profiles
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+
+[[create-multi-instances-of-minishift]]
+== Create multiple instances of Minishift
+
+With the help of profile feature you can create multiple instances of Minishift with xref:../command-ref/minishift_start.adoc#[`minishift start --profile`] command.
```

Comment:
"of the profile feature"
"with" -> "using"

Reply:
So we still need something before the 'Create' part as an intro

Reply:
I am working on it.

Reply:
Drop 'help of'. => 'With the profile feature'